.travis.yml

```diff
@@ -1,4 +1,5 @@
 language: c++
+dist: xenial
```

Comment:
This is kind of tricky, we use the older distro on purpose: we build releases on an ancient version for compatibility reasons. So currently we run CI on the same platform that we actually build releases with.

Reply:
Let me rollback this one!